### DISCUSSION TITLE
NewType：会思考的输入法，基于 Whisper / 豆包 的语音键盘

### DISCUSSION THREAD

**Discussion by Gloridust** [creatorsand/.github#77] [Ideas]
**一句话 IDEA**：做一个会思考的输入法，基于 Whisper / 豆包 的语音键盘，能处理语句当中的停顿、语气词等，还可以一键润色。

**我的角色**：产品经理 / 后端 / Swift 开发者；

**其他角色需求**（不包含发起人）：Swift 开发者，会 Git 的基本操作即可；

**社群昵称**：Ethan Zou
**个人简介**：大数据专业学生 | 前端/后端/移动开发/提示词工程师 ｜ 学习 React/NextJS, Flutter/Python, 机器学习和大数据 | YGeeker 联合创始人
**个人博客**： [Ethan Zou's Blog](https://gloridust.xyz/)
**Github**：[Github Profile](https://github.com/Gloridust)

**用户群体**：日常在 IM APP 沟通频繁，手动打字慢，市面上的语音输入法不能解决痛点；

**Why me**：用户为什么选择我们的产品？
1. 大多数 iOS 用户对于传统第三方键盘是没有需求的，原生输入法即最佳解（语音输入除外），所以没必要做“一整个”输入法（用户只会使用语音功能）；
2. AI 是我们的主角，而不是配角。传统厂商的语音输入法只是放在空格键上的一个附加功能，而我们的产品以“语音输入”为主角，可以围绕语音功能放下更多的自定义选项、快捷键等；

**竞品优势**：大船难掉头。百度输入法不可能调用豆包的模型，即使他们自己的模型做的很菜。我们作为“局外人”--一个第三方小团队，能够合理运用国内外各个厂商模型的优势，快速决策迭代。

**Comment by Gloridust** [comment]
熟悉 Swift 和Apple 生态的朋友还可以帮忙看看现在有没有办法本地跑 Whisper，这样就能保证 Local-First 了～

**Comment by ZigaoWang** [comment]
我不太确定，但是 github 上应该有很多开源的，例如这个我随便找的：
https://github.com/jianchang512/stt

**Reply by Gloridust** [reply]
感谢☺️，但是大多数都是跑在PC/Mac端的，暂时没发现能跑在iOS/iPadOS的端侧ASR模型😢
